src/uriHandler.ts

```diff
@@ -25,17 +26,19 @@ export class UriEventHandler extends vscode.EventEmitter<vscode.Uri> implements
     return UriEventHandler.instance;
   }
 
-  public handleUri(uri: vscode.Uri) {
-    switch (uri.path) {
+  public async handleUri(uri: vscode.Uri) {
+    const { path } = uri;
+    switch (path) {
       case "/authCallback":
         logger.debug("Got authCallback URI, firing as Event", uri);
         this.fire(uri);
         break;
       case "/consume":
         vscode.commands.executeCommand("confluent.topic.consume.fromUri", uri);
         break;
-      default:
-        logger.warn("Got unexpected URI, ignoring", uri);
+      case "/projectScaffold":
+        vscode.window.showErrorMessage("Unsupported URI path.");
```

Comment:
Looks like we lost `projectScaffoldUri.fire(uri);` from before -- a little too much cleanup!

Reply:
eek! just added back in and pushed

Reply:
getting some issues with local setup on testing though... 
```
Running extension tests from the command line is currently only supported if no other instance of Code is running.
```

gotta hunt down where I'm running this